crates/symbolicator-service/src/services/objects/mod.rs

```diff
@@ -254,19 +254,31 @@ fn create_candidates(sources: &[SourceConfig], lookups: &[FoundMeta]) -> AllObje
     for meta_lookup in lookups.iter() {
         let source_id = meta_lookup.file_source.source_id();
         source_ids.take(source_id);
-        candidates.push(create_candidate_info(meta_lookup));
+        let location = meta_lookup.file_source.uri();
+        if let Err(idx) =
+            candidates.binary_search_by_key(&(source_id, &location), |c| (&c.source, &c.location))
```

Comment:
I’m a bit surprised this code here needs special attention. In which cases does the `lookups` contain duplicates?

Reply:
Probably a good idea to find out why.